metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -329,6 +329,9 @@ def _visit(node, workflow, exit_node=None):
     def _cron(self):
         schedule = self.flow._flow_decorators.get("schedule")
         if schedule:
+            assert (
```

Comment:
rather than the assert, can you raise `StepFunctionsException` so that Metaflow can pretty print the exception?

Reply:
Done.